_posts/2021-11-05-eslint-v8.2.0-released.md

```diff
@@ -1,6 +1,9 @@
 ---
 layout: post
 title: ESLint v8.2.0 released
+author: mdjermanovic
+categories:
+  - Release Notes
```

Comment:
This is something we're not using yet?

Reply:
Correct. I need to update the template. The new blog design supports categories to help group posts together.